compiler/ballerina-parser/src/main/java/io/ballerinalang/compiler/internal/parser/BallerinaParser.java

```diff
@@ -10179,6 +10188,66 @@ private STNode parseOuterKeyword() {
         }
     }
 
+    /**
+     * Parse order-keyword.
+     *
+     * @return Order-keyword node
+     */
+    private STNode parseOrderKeyword() {
```

Comment:
Need to update `resumeParsing` method with each of these newly added methods (if they are calling the `recover` method)